### PR TITLE
fix rcnn eval on voc

### PR DESCRIPTION
**Pull request by jerrywgz**
Fix lack of gt_box and gt_label in RCNN eval on VOC dataset

### CODE REVIEW

File: PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py [hunk 1]
```diff
@@ -90,6 +90,46 @@ def __call__(self, sample, context=None):
         return outs
 
 
+@register_op
+class ArrangeEvalRCNN(BaseOperator):
+    """
+    Transform dict to the tuple format needed for training.
```
Comment (heavengate) [inline PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py]: training -> evaluation

File: PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py [hunk 2]
```diff
@@ -90,6 +90,46 @@ def __call__(self, sample, context=None):
...
+
+    def __call__(self, sample, context=None):
+        """
+        Args:
+            sample: a dict which contains image
+                    info and annotation info.
+            context: a dict which contains additional info.
+        Returns:
+            sample: a tuple containing the following items:
+                    (image, im_info, im_id)
```
Comment (heavengate) [inline PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py]: (im, im_info, im_id, im_shape, gt_bbox, gt_class, difficult)

Comment (jerrywgz) [reply PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py]: Done

File: PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py (hunk 1 above)
Comment (jerrywgz) [reply PaddleCV/PaddleDetection/ppdet/data/transform/arrange_sample.py]: Done

**Review by heavengate — APPROVED** [review]

**Pull request merged**